recipes/hdmedians/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "hdmedians" %}
+{% set version = "0.13" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 230f80e064d905c49a1941af1b7e806e2f22b3c9a90ad5c21fd17d72636ea277
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython >=0.23
+    - numpy
+    - pip
+    - python
+    - nose
+  run:
+    - numpy
+    - python
+test:
+  imports:
+    - hdmedians
+
+about:
+  home: http://github.com/daleroberts/hdmedians
+  license: GPL3
+  license_family: GPL3
```

Comment:
Since this is GPL'd you need to include the `license_file` entry here.  If the license does not come in the tarball, feel free to stick a copy of it in the recipe: https://conda-forge.org/docs/meta.html#packaging-the-license-manually